### PR TITLE
CC and MIT license inclusion

### PR DESCRIPTION
**Pull request by PeteBoucher**
[Pivotal Chore 67910808](https://www.pivotaltracker.com/story/show/67910808) Terms of Service and Privacy Policy on website
- Removes restrictive boilerplate copyright notice
- Adds Creative Commons Attribution license in context of site content
- Adds MIT license in context of source code by rendering the `LICENSE` file so that this can be altered in-place in future

### CODE REVIEW

**Comment by yggie** [comment]
I was speaking to @tochman before he went off, I think he was looking for a slightly different open source license, we will need to talk about this some time, but essentially he wants it to be open source but we specify the way in which the works are attributed, what are your thoughts? This needs a little more work before it can be merged, we should keep it copyright till we figure this out.

**Comment by PeteBoucher** [comment]
Do you mean to change the MIT open source software license (which is very permissive) or the Creative Commons Attribution content license? These apply to different parts of the project.

**Comment by yggie** [comment]
Yes I think we discussed only the source code. Creative Commons should be good for the other assets.

**Comment by PeteBoucher** [comment]
Ok this is something we need to discuss. The first question seems to be; do we want to allow organisations (like Hatch) to use our code in a commercial product? Where they charge students to participate in a class that makes use of our IDE and student management software.

If the answer is yes, which I think we have determined it is. Then I think that rules out the GPL and LGPL family of licenses. http://blog.codinghorror.com/pick-a-license-any-license/

**Comment by yggie** [comment]
Thats an awesome article, I have that bookmarked myself :smile: we want it to be as permissive as MIT but with attribution compulsory and in the way we specify, I will have a closer look at this when I have time this weekend.

**Comment by yggie** [comment]
On closer inspection, I think BSD and Apache comes pretty close to our requirements, will need to dig deeper though...

**Comment by yggie** [comment]
@PeteBoucher unfortunately the 3-clause version of BSD does not include the clause specifying the manner of attribution, I was thinking of using the 4-clause version [here](https://en.wikipedia.org/wiki/BSD_licenses#4-clause_license_.28original_.22BSD_License.22.29)

**Comment by yggie** [comment]
@PeteBoucher since we haven't decided on what to do with the open source license (BSD didn't fit all our needs), could we split this up so we can have the CC merged in independently?

File: app/views/static_pages/terms_and_conditions.html.erb [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+    <li>Share — copy and redistribute the material in any medium or format</li>
+    <li>Adapt — remix, transform, and build upon the material</li>
+  </ul>
+  for any purpose, even commercially. Under the folowing terms:</p>
+  <p><strong>Attribution</strong><br />You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.</p>
+
+  <p><strong>Intellectual Property: Source code</strong><br />
+    The Site and its features, functionality and source code are owned by AGILEVENTURES NONPROFIT LTD. and are licenced under the <a href="http://opensource.org/licenses/MIT">MIT</a> open source license:</p>
+  <blockquote>
+    <%= render file: 'LICENSE' %>
```
Comment (yggie) [inline app/views/static_pages/terms_and_conditions.html.erb:19]: :+1:

**Comment by yggie** [comment]
@PeteBoucher looks great, do you mind squashing the commits? This should really be a one commit thing

**Pull request merged**